config.json

```diff
@@ -11,6 +11,12 @@
 
     // Mode in which the dashboard server express environment runs
     NODE_ENV: development
+    // Base URL pattern to use when generating public links
+    // Supported placeholder values:
```

Comment:
Needs more detail about where these values come from. Something like "The values are populated from the Host request header when a notebook is POSTed to the server."